_docs/2.0.0/usage/errcol.md

```diff
@@ -1,25 +1,22 @@
 ---
 layout: docs
-title: Usage - errCol
+title: Usage - Error Column
 version: '2.0.0'
 categories:
     - '2.0.0'
     - usage
 redirect_from: /docs/usage/errcol
 ---
-
-Error Column
-============
-
+## Table Of Content
 <!-- toc -->
-- [Error Column](#error-column)
-  - [Description](#description)
-  - [Error Types](#error-types)
-  - [Notes](#notes)
+- [Table Of Content](#table-of-content)
+- [Description](#description)
+- [Error Types](#error-types)
+- [Notes](#notes)
 <!-- tocstop -->
 
-Description
------------
+## Description
```

Comment:
Why changing the style of headers?

Reply:
https://cirosantilli.com/markdown-style-guide#headers

multiple markdown style guides say not to mix header types. Some even prohibit it completely as it can break some markdown engines (usually found in chats)

Reply:
Subjectively to keep it all in one style

Reply:
Also, H1 heading is not needed as it is the title of the page.

Reply:
Fair enough.